website/pages/docs/reference/source-spec.md

```diff
@@ -1,6 +1,8 @@
 # Source Spec Reference
 
-This goes through all the available options for the source plugin `spec` object:
+Following are available options for the top level source plugin `spec` object. 
+
+Note: For configuring individual plugins, please refer to the configuration section from the relevant plugins from [here](https://www.cloudquery.io/docs/plugins/sources). (eg. [AWS plugin configuration](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/aws/docs/configuration.md)). Also checkout our [recipes](https://www.cloudquery.io/docs/recipes/aws-postgresql) for common configuration examples.
```

Comment:
```suggestion
Note: For configuring individual plugins, please refer to the configuration section from the relevant plugins from [here](https://www.cloudquery.io/docs/plugins/sources). (eg. [AWS plugin configuration](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/aws/docs/configuration.md)). Also check out our [recipes](https://www.cloudquery.io/docs/recipes/aws-postgresql) for common configuration examples.
```

Not sure about this, but Google says this:
![image](https://user-images.githubusercontent.com/26760571/195616783-72cc60ba-a9d1-4f12-b339-29dee0073469.png)